src/main/java/software/amazon/awssdk/crt/mqtt5/packets/UnsubscribePacket.java

```diff
@@ -80,6 +80,15 @@ public UnsubscribePacketBuilder withUserProperties(List<UserProperty> userProper
          */
         public UnsubscribePacketBuilder() {}
 
+        /**
+         * Creates a new UnsubscribePacketBuilder with one subscription defined.
+         *
+         * @param subscription A single topic filter that the client wishes to unsubscribe from
+         */
+        public UnsubscribePacketBuilder(String subscription) {
```

Comment:
topicFilter instead of subscription

Reply:
I renamed the parameter in `withSubscription` as well